docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -607,6 +786,61 @@ The `locale` prop of the `LocalizationProvider` component have been renamed `ada
    />
   ```
 
+### Input
+
+- The `InputProps` prop has been removed.
+  You can use the `InputProps` of the `input` component props slot instead:
+
+  ```diff
+   <DatePicker
+  -  InputProps={{ color: 'primary' }}
+  +  componentsProps={{ input: { InputProps: { color: 'primary' }}}}
+   />
+  ```
+
+  :::warning
+  This change only applies to the new pickers.
+  The legacy pickers keep the `InputProps` prop until there removal.
+
+  For more information about those new pickers, take a look at the [New picker components](#new-picker-components) section
+  :::
+
+### Input adornment
+
+- The `InputAdornmentProps` prop has been replaced by a `inputAdornment` component props slot:
```

Comment:
General nitpick: The prop is named `componentsProps`, but in the most part of the documentation we are calling it `component props` instead of `components props`. Maybe it would be worth keeping those more in sync between the prop name and usage? 🤔

Reply:
So you would work it `components props slot`, make sense to me

Reply:
Hm. Technically, I'd consider `slot` a place where I can put/replace something. 🤔 
In case of `components props` it's not really a slot.
What do you think about using `components slot(s)` and `components props` in the documentation as well?
Basically, drop `slot` where we are talking about props and not the component slot itself...

Reply:
`component slots props` ?
When we will rename the prop `slotsProps`, won't it be weird if we don't have the word `slot` at all in the sentence ?

Reply:
Fair point. 🤔 
I thought that we could change these at the same time when we move towards `slots` and `slotsProps`, but on the other hand, just sticking to `component slots props` probably seems the best...
I do agree with your suggestion. 👍

Reply:
Applied